spec/lib/mi_bridges/driver/pages_spec.rb

```diff
@@ -17,4 +17,28 @@
       end
     end
   end
+
+  context "when instantiating a page too many times" do
+    it "raises an exception" do
+      snap_application = double
+
+      expect do
+        6.times { MiBridges::Driver::BasePage.new(snap_application) }
```

Comment:
would be good to reference a constant or var here so `6` feels less like a magic number